docs/reference/eql/syntax.asciidoc

```diff
@@ -0,0 +1,224 @@
+[role="xpack"]
+[testenv="basic"]
+[[eql-syntax]]
+== EQL syntax reference
+
+experimental::[]
+
+[IMPORTANT]
+====
+{es} supports a subset of EQL syntax.
+====
+
+[discrete]
+[[eql-basic-syntax]]
+=== Basic syntax
+
+EQL queries require an event type and a matching condition. The `where` keyword connects them.
+
+[source,eql]
+----
+event_type where condition
+----
+
+For example, the following EQL query matches `process` events with a `name`
+field value of `svchost.exe`:
+
+[source,eql]
+----
+process where name == "svchost.exe"
```

Comment:
not sure if this was intentional for the sake of simplicity, but if we do want to make these examples more ECS-compliant:
```suggestion
process where process.name == "svchost.exe"
```